src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt

```diff
@@ -28,8 +28,7 @@ object CustomNbtEvents: PaperBinFeature {
 	}
 	
 	override fun registerTransformers() {
-		return
-		register("net.minecraft.server.v1_12_R1.Entity") { classNode ->
+		return register("net.minecraft.server.v1_12_R1.Entity") { classNode ->

```

Comment:
The previous formatting was intentional and has a different meaning to the new formatting.

Reply:
Alright, thought it was a typo

Reply:
The transformer is not being used so the `return` was to exit it before doing anything.
New format would still run the `register` body.

Reply:
👍